src/vcpkg/buildenvironment.cpp

```diff
@@ -17,6 +17,8 @@ namespace vcpkg
         local_variables.emplace_back("BUILDTREES_DIR", paths.buildtrees());
         local_variables.emplace_back("_VCPKG_INSTALLED_DIR", paths.installed().root());
         local_variables.emplace_back("DOWNLOADS", paths.downloads);
+        local_variables.emplace_back("VCPKG_SCRIPTS_DIR", paths.scripts);
```

Comment:
Should this be just `SCRIPTS` since it is already defined in `ports.cmake` ?

Reply:
That makes sense to me